repobee_sanitizer/_sanitize_repo.py

```diff
@@ -136,7 +140,11 @@ def _git_commit_on_branch(repo_root: pathlib.Path, target_branch: str):
     repo = git.Repo(str(repo_root))
     repo.git.symbolic_ref("HEAD", f"refs/heads/{target_branch}")
     repo.git.add(".", "--force")
-    repo.git.commit("-m", "'Sanitize files'")
+    try:
+        repo.git.commit("-m", "'Sanitize files'")
+    except git.GitCommandError as exc:
+        if "nothing to commit, working tree clean" in exc.__str__():
+            raise EmptyCommitError()
```

Comment:
Always do `raise from` to include the whole stacktrace.

```suggestion
            raise EmptyCommitError() from exc
```